lib/address_geocoder.rb

```diff
@@ -56,7 +56,7 @@ def call_google
         if attempts <= 5
           retry
         else
-          raise ConnectionError, 'Could not connect to GoogleAPI'
+          raise SystemCallError, 'Could not connect to GoogleAPI'
```

Comment:
you mean just `raise` without argument?
according to following link,
https://rubymonk.com/learning/books/4-ruby-primer-ascent/chapters/41-exceptions/lessons/92-handling#275
```
To raise exceptions we use raise or Kernel.raise or Kernel.fail. raise without any arguments simply raises a RuntimeError exception with the string message that we've specified.
```